problem/tools/settings/unittest.cc

```diff
@@ -0,0 +1,133 @@
+#include <gmock/gmock.h>
+#include <gtest/gtest.h>
+#include "../cppaudit/gtest_ext.h"
+
+// #include <sstream>
+#include <vector>
+#include "../../game.h"
+
+using ::testing::HasSubstr;
+// Include the header files from the student solution.
+// Use ../../ prefix to locate the student solution folder
+// For example:
+// #include "../../math.h"
+
+/* Sample Unit test
+
+TEST(Factorial, Zero) {
+	ASSERT_EQ(factorial(0), 1);
+}
+
+*/
+// TESTS
```

Comment:
update the test comments to represent what they actually do instead of repeating the unittest name